codecov_auth/admin.py

```diff
@@ -111,6 +113,40 @@ def impersonate_owner(self, request, queryset):
 impersonate_owner.short_description = "Impersonate the selected owner"
 
 
+def refresh_owner(self, request, queryset):
+    if queryset.count() != 1:
+        self.message_user(
+            request, "You must refresh exactly one Owner.", level=messages.ERROR
+        )
+        return
+
+    owner = queryset.first()
+
+    task_service = TaskService()
+    task_service.refresh(
+        ownerid=owner.ownerid,
+        username=owner.username,
+        sync_teams=True,
+        sync_repos=True,
+        manual_trigger=True,
+    )
+
+    self.message_user(
+        request,
+        f"Refresh task triggered for {owner.username} (ownerid: {owner.ownerid})",
+        level=messages.SUCCESS,
+    )
+    History.log(
+        Owner.objects.get(ownerid=owner.ownerid),
```

Comment:
this would just be `owner`, right?

Reply:
SO I copied what `impersonate_owner` does :P